helm_deploy/create-and-vary-a-licence-api/values.yaml

```diff
@@ -37,6 +37,36 @@ generic-service:
       DB_NAME: "database_name"
       DB_USER: "database_username"
       DB_PASS: "database_password"
+    notify-templates:
+      NOTIFY_TEMPLATES_VARIATIONFORAPPROVAL: "VARIATION_FOR_APPROVAL"
+      NOTIFY_TEMPLATES_LICENCEAPPROVED: "LICENCE_APPROVED"
+      NOTIFY_TEMPLATES_INITIALLICENCEPROMPT: "INITIAL_LICENCE_PROMPT"
+      NOTIFY_TEMPLATES_URGENTLICENCEPROMPT: "URGENT_LICENCE_PROMPT"
+    pdu-head-config:
+      PDU_CONTACTS_WPTNWS_FORENAME: "WPTNWS_FORENAME"
```

Comment:
Whats the plan for after pilot? Sorry if I've not followed conversation - this wont scale well when we roll out to the nation?

Reply:
Community API will have a call to retrieve the ACO level staff in a PDU.